private/buf/bufgen/features.go

```diff
@@ -85,6 +86,7 @@ func computeRequiredFeatures(image bufimage.Image) *requiredFeatures {
 }
 
 func checkRequiredFeatures(
+	container appext.LoggerContainer,
```

Comment:
Just pass in a `*zap.Logger` that you got from `NewGenerator` and revert the change to `appext.Container`